tests/cpp/test_id_model.cpp

```diff
@@ -42,7 +42,7 @@ TEST_F(IdModelTest, DetectSelfMapping) {
   EXPECT_THAT(
       [&]() { IdModel id_model(&fusion, /*build_graphs=*/true); },
       ::testing::ThrowsMessage<nvfuser::nvfError>(
-          ::testing::HasSubstr("!hasSelfMapping")));
+          ::testing::HasSubstr("are mapped with each other")));
```

Comment:
This was supposed to detect an error message coming from IdModel but it was actually we were detecting an error message by ComputeAtMap when the validation was done. Now that the validation is disabled, the error message is indeed from IdModel, which has a different error message.